media-libs/libsdl3/libsdl3-3.2.4.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Simple Direct Media Layer 3.0"
+DESCRIPTION="Simple DirectMedia Layer is a cross-platform development library \
+designed to provide low level access to audio, keyboard, mouse, joystick, and \
+graphics hardware via OpenGL and Direct3D. It is used by video playback \
+software, emulators, and popular games."
+HOMEPAGE="https://www.libsdl.org/"
+COPYRIGHT="1997-2025 Sam Lantinga"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://www.libsdl.org/release/SDL3-$portVersion.tar.gz"
+CHECKSUM_SHA256="2938328317301dfbe30176d79c251733aa5e7ec5c436c800b99ed4da7adcb0f0"
+SOURCE_DIR="SDL3-$portVersion"
+# PATCHES=""
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
```

Comment:
According to @Begasus in https://github.com/haikuports/haikuports/pull/11770#discussion_r1952040426, this should be `"0.2.4"`.

Reply:
Sorry, stupid mistake.